README.md

```diff
@@ -1 +1,3 @@
-# To-Do-List-Best-Practices
\ No newline at end of file
+# To-Do-List-Best-Practices
+
+A practice javascript exercise to pass reviews between 2 coders and give recommendations based on their evaluations and assumptions.
```

Comment:
- Please kindly update your readme file by adding project name and description of the project